Allow Rails 7.2

texpert opened the pull request:
Add `7-2-stable` Rails branch to the CI

mvz commented on .github/workflows/ruby.yml:
```diff
@@ -21,7 +21,7 @@ jobs:
       matrix:
         ruby: ["3.0", "3.1", "3.2", "3.3"]
-        appraisal: [rails_6_0, rails_6_1, rails_7_0, rails_7_1]
+        appraisal: [rails_6_0, rails_6_1, rails_7_0, rails_7_1, rails_7_2]
```
From the test failures, it looks like rails 7.2 will not support Ruby 3.0. Can you please exclude that combination from the matrix?

mvz commented on non-digest-assets.gemspec:
```diff
@@ -22,13 +22,13 @@ Gem::Specification.new do |spec|
   spec.metadata["rubygems_mfa_required"] = "true"
 
-  spec.add_dependency "activesupport", [">= 6.0", "< 7.2"]
+  spec.add_dependency "activesupport", [">= 6.0", "<= 7.2"]
```
This will exclude activesupport 7.2.1 and up. Please change it to `< 7.3`.

mvz commented on non-digest-assets.gemspec:
```diff
@@ -22,13 +22,13 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "aruba", "~> 2.0"
   spec.add_development_dependency "pry", "~> 0.14.0"
-  spec.add_development_dependency "rails", [">= 6.0", "< 7.2"]
+  spec.add_development_dependency "rails", [">= 6.0", "<= 7.2"]
```
This will exclude rails 7.2.1 and up. Please change it to `< 7.3`.

mvz commented on gemfiles/rails_7_2.gemfile:
```diff
@@ -0,0 +1,11 @@
+
+source "https://rubygems.org"
+
+gem "rails", git: 'https://github.com/rails/rails', branch: '7-2-stable'
```
Please fix the RuboCop offenses on this line.

mvz reviewed (changes requested):
Thanks for this @texpert! If you can take care of my comments this should be good to merge.

texpert commented:
> Thanks for this @texpert! If you can take care of my comments this should be good to merge.

Thanks for review, @mvz! I have addressed all the comments.

mvz commented:
Thanks again, @texpert!

texpert commented:
Thanks for help, @mvz!

Can you, please, craft a new release?!

mvz commented:
Hi @texpert I just released version 2.4.0 which includes this change.
